src/Presentation/Console/Command/QaAutomationCommand.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Presentation\Console\Command;
+
+use Doctrine\DBAL\Exception\TableNotFoundException;
+use SprykerSdk\Sdk\Core\Application\Dependency\ContextRepositoryInterface;
+use SprykerSdk\Sdk\Core\Application\Dependency\ProjectSettingRepositoryInterface;
+use SprykerSdk\Sdk\Core\Application\Dependency\Repository\TaskRepositoryInterface;
+use SprykerSdk\Sdk\Core\Application\Service\ContextFactory;
+use SprykerSdk\Sdk\Core\Application\Service\ProjectWorkflow;
+use SprykerSdk\Sdk\Core\Application\Service\TaskExecutor;
+use SprykerSdk\Sdk\Core\Domain\Entity\Command;
+use SprykerSdk\Sdk\Core\Domain\Entity\Lifecycle\InitializedEventData;
+use SprykerSdk\Sdk\Core\Domain\Entity\Lifecycle\Lifecycle;
+use SprykerSdk\Sdk\Core\Domain\Entity\Lifecycle\RemovedEventData;
+use SprykerSdk\Sdk\Core\Domain\Entity\Lifecycle\UpdatedEventData;
+use SprykerSdk\Sdk\Core\Domain\Entity\Task;
+use SprykerSdk\SdkContracts\Entity\ContextInterface;
+use SprykerSdk\SdkContracts\Entity\ErrorCommandInterface;
+use SprykerSdk\SdkContracts\Entity\ExecutableCommandInterface;
+use SprykerSdk\SdkContracts\Entity\StagedTaskInterface;
+use SprykerSdk\SdkContracts\Entity\TaskInterface;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class QaAutomationCommand extends RunTaskWrapperCommand
+{
+    /**
+     * @var string
+     */
+    protected const TASKS_SETTING_KEY = 'qa_tasks';
+
+    /**
+     * @var string
+     */
+    protected const DESCRIPTION = 'Run configurable qa tasks.';
+
+    /**
+     * @var string
+     */
+    protected const COMMAND_NAME = 'qa:run';
```

Comment:
should be `sdk:qa:run` according to our command namig rules.

Reply:
fixed